HEAnalytics/HEAnalyticsUser.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  HEAnalyticsUser.swift
+//
+//  Created by Ben Kreeger on 3/17/16.
```

Comment:
You can throw in your own contact info (e.g. email or whatever) if you wish. No requirement to.